src/devtools/devtools.tsx

```diff
@@ -204,13 +224,13 @@ export function ReactQueryDevtools({
     <Container ref={rootRef} className="ReactQueryDevtools">
       <ThemeProvider theme={theme}>
         <ReactQueryDevtoolsPanel
-          ref={panelRef}
+          ref={panelRef as any}
```

Comment:
Had to use `any` here because it was giving a weird error -
```
Type 'LegacyRef<HTMLDivElement>' is not assignable to type 'Ref<HTMLDivElement> | undefined'.
```

I tried looking for the solution but couldn't find anything

Reply:
yeah, me neither. I guess it's fine for now 🤷